misk-hibernate/src/test/kotlin/misk/hibernate/ReflectionQueryFactoryTest.kt

```diff
@@ -534,12 +786,13 @@ class ReflectionQueryFactoryTest {
     }
 
     // count beyond max results
-    transacter.transaction { session ->
-      assertThat(queryFactory.newQuery<OperatorsMovieQuery>()
-        .allowFullScatter().allowTableScan()
-        .apply { maxRows = 2 }
-        .count(session))
-        .isEqualTo(3)
+    assertFailsWith<java.lang.IllegalStateException> {
```

Comment:
is this a backwards in-compatible change? Is it safe to make?

Reply:
@swankjesse what do you think. I think it is safe enough. I guess someone could be adding a maxrows to a count query. However since this was never actually used all that needs to happen is remove that attribute.

Reply:
I think so

Reply:
(And if we do find crashes, they’re likely bugs)